considerations/minutes/economic-cab/2022-10-29-economic-cab-minutes-sip-15.md

```diff
@@ -0,0 +1,66 @@
+# Economic CAB Minutes
+
+## Meeting Information
+
+**Location:** Google Meet
+
+**Materials**: [Sip-015 proposal](https://github.com/stacksgov/sips/blob/feat/sip-015/sips/sip-015/sip-015-network-upgrade.md)
+
+**Date:** Oct 29 2022
+
+**Time:** 1pm ET
```

Comment:
Can you put this in UTC?

Reply:
Did the conversion, this should be the update below assuming the meeting was scheduled for 1hr:

**Time:** 17:00 - 18:00 UTC